### ISSUE TITLE
[ENHANCEMENT]: automatically shift compensation curve after DHW cycle

### ISSUE THREAD

**Issue by root2k** [edterbak/NodeRed_Heishamon_control#90]
**Is your feature request related to a problem or use case? Please describe.**
Since I have the pana, I have allways the same problem after the heatpump finished DHW cycle.
The compressor goes down to 0 hz and the heatpump isn't heating for hours.
That's a really disapointing part of the panasonic control because when the outside temperature is really cold and the heatpump isn't heating after DHW cycle, the house cools down.
I'm using the timer of the remote control right now, but it isn't really comfortable because you are limited to just 6 timers.

**Describe the solution you'd like**
It would be very helpful for me, and maybe some others, to have a feature to shift up the compensation curve automatically up to 3 for a specified time after DHW cycle is finished.

The feature needs recognize if the three-way-valve switches from tank to room.
If this is happened and the compressor is going to zero the automatic should be started for the duration of ... minutes.
But what I noticed so far watching this problem, when I shift up to +3 and going directly back to 0 after ... minutes the compressor shuts down again. That's because of the bahvior of the panasonic control.
So to impede this, I need to do the follwing: +3 for 10 min, +2 for 10 min, +1 for 10 min. and then back to 0.

So if it is possible the automatic should do the same thing.

**Describe alternatives you've considered**
In case of the duration of this automatic I'm not sure but in my case some secound are not sufficient. I think 5 min. or something like that would be good.
But the duration can be adjusted manually by each user in the flows I think?
Or a nice to have is, that each user can set the duration by itself.

**Comment by root2k** [comment]
No definately not. That's not a problem of my panasonic.
I'm quite active in an german board, where are lots of people suffering with this problem.
Some people created a home automation with iobroker or something else to solve this issue.

**Comment by MikeyMan83** [comment]
Ok, but why am I not seeing this issue then?

**Comment by edterbak** [comment]
It has been discussed in other topic better. see this.
https://github.com/edterbak/NodeRed_Heishamon_control/issues/64

I know what root2k requests.

Issue he is facing is due to the fact that sometimes after a dhw run, the water temperature is too high for the pump to start the compressor already. only after the system has lost x amount of heat, the water temperature is low enough that it will start the compressor. 
By just giving the pump a little nudge in setpoint for a minute or so, (optional ofc) the pump starts faster and temperature is more stable.

**Comment by root2k** [comment]
That's excactly what I mean, could'nt have described it better. Would be nice to have in the next releases ...

Thank's edterbak (and others involved in this project) for your effort and time ...

**Comment by edterbak** [comment]
Just reading again this issue.

Have you played with this setting?
![2023-09-03 21_44_45-Panasonic Dashboard](https://github.com/edterbak/NodeRed_Heishamon_control/assets/3155621/02dfabe3-4d70-4faf-9d78-a5a65ad41f12)

![2023-09-03 21_45_04-Panasonic Dashboard](https://github.com/edterbak/NodeRed_Heishamon_control/assets/3155621/ef0559f6-5594-4269-92b3-e4fa21b7e57b)

Increasing this value from +2 to a value like +10, would that help in your case? Currently it is limited to +10, but I can increase that if needed. 

What this value does; it is the upper limit for the allowed setpoint temperature. This is relative to the starting setpoint. temperature. The SoftStart function will try to stick to 1 degree below the measured water temperature. For this correction, you can set a upper and lower limit where it is allowed to follow the actual water temperature.
From moment 0 - 3 minutes, there is a hardcoded limit to increase the setpoint to +7 compared to the starting SP

From moment 3 - xxx minutes, This value can be used. Default is 2, but you can increase it.

Does this help in anyway?
